tests/integration/fixtures/mini_sentry.py

```diff
@@ -41,8 +41,7 @@ def __init__(self, server_address, app):
         self.captured_events = Queue()
         self.captured_outcomes = Queue()
         self.captured_metrics = Queue()
-        self.test_failures = []
-        self.reraise_test_failures = True
```

Comment:
I introduced this field in a previous PR, but did not realize we already have `fail_on_relay_error`.